website/docs/configuration/variables.html.md

```diff
@@ -452,8 +452,8 @@ precedence over earlier ones:
 * Any `*.auto.tfvars` or `*.auto.tfvars.json` files, processed in lexical order
   of their filenames.
 * Any `-var` and `-var-file` options on the command line, in the order they
-  are provided. (This includes variables set by a Terraform Cloud
-  workspace.)
+  are provided.  
+* Variables set by a Terraform Cloud or Enterprise workspace.
```

Comment:
The truth of it is that this clean ordering is actually dependent on your [Execution Mode](https://www.terraform.io/docs/cloud/workspaces/settings.html#execution-mode), when Terraform Cloud is involved. Using the remote backend with [remote operations](https://www.terraform.io/docs/cloud/run/index.html), I don't believe `terraform.tfvars` is respected at all and `-var` is explicitly not supported (this, coincidentally, is exactly how Terraform Cloud provides workspace variables - which is probably why it was on the same line in the original version of this change). 

Conversely, if a Terraform Cloud workspace is being used only as a [state storage backend](https://www.terraform.io/docs/state/remote.html), the workspace variables configured should not be applicable, either.

Reply:
Yeah, I think I was the one who put TFC vars on the same precedence line as `-var-file`, and that was exactly why. That probably pre-dated being able to use the `remote` backend with local execution mode, though, because that definitely complicates the issue. HMM. 

Probably this section needs an overhaul, not just a one-line tweak. Given the pending split between CLI and language docs, I think the right move is to add a small page to the CLI docs about providing variable values (in the "provisioning" section of the sidebar, probably), and then remove most of this section and link out to both the TF CLI and TFC docs about that.